packages/oid4vci-demo-frontend/public/locales/development/en/translation.json

```diff
@@ -66,6 +67,8 @@
   "ssi_information_request_page_form_tax_registration_number_label": "TRN",
   "ssi_information_request_page_form_contact_first_name_label": "Contact first name",
   "ssi_information_request_page_form_contact_last_name_label": "Contact last name",
+  "ssi_information_request_page_form_contact_bsn_label": "Contact BSN",
+  "ssi_information_request_page_form_tax_registration_code_label": "Tax Registration code",
```

Comment:
ssi_information_request_page_form_tax_registration_number_label

Reply:
that's TRN and not the full name. based on the design it should be the full name

Reply:
Check with Maarten, because then we should change it on all places, meaning we still have duplicates

Reply:
following our discussion, changed it